### DISCUSSION TITLE
How to use SDXL Inpainting model to remove objects?

### DISCUSSION THREAD

**Discussion by ammarft-ai** [huggingface/diffusers#9828] [Q&A]
Hi, I am using inpainting models for editing and adding new objects for a while but I can't figure out how to remove a masked object from the image. I tried giving empty prompt but it didn't work as well.

**Comment by ukaprch** [comment]
If I understand you correctly removing an object would not necessarily be inpainting in the sense that you don't use a text prompt, but only a mask. For this you would use segment anything to obtain your mask and lama inpainting to remove the object.
see here for more details: [https://github.com/geekyutao/Inpaint-Anything](url)

**Comment by yiyixuxu** [comment]
@asomoza has a guide on remove object here https://github.com/huggingface/diffusers/discussions/9433
